main.js

```diff
@@ -0,0 +1,16 @@
+$('#submit').on('click', function () {
+  var textInput = $('#message').val();
```

Comment:
Chaining jQuery methods this way is fine, but it's not the most resilient option - what would happen if your DOM changes? It might be good to consider other options.